src/main/java/hudson/plugins/emailext/ExtendedEmailPublisher.java

```diff
@@ -487,6 +507,58 @@ private boolean executePresendScript(ExtendedEmailPublisherContext context, Mime
         return !cancel;
     }
 
+    private void executePostsendScript(ExtendedEmailPublisherContext context, MimeMessage msg, Session session, Transport transport)
```

Comment:
Could the executePresendScript just be updated to be more generic and be called executeScript?